code/modules/cargo/order.dm

```diff
@@ -61,8 +61,8 @@
 	P.update_appearance()
 	return P
 
-/datum/supply_order/proc/generateManifest(obj/container, owner, packname) //generates-the-manifests.
-	var/obj/item/paper/fluff/jobs/cargo/manifest/P = new(container, id, 0)
+/datum/supply_order/proc/generateManifest(obj/container, owner, packname, cost_override) //generates-the-manifests.
+	var/obj/item/paper/fluff/jobs/cargo/manifest/P = new(container, id, cost_override || pack.cost)
```

Comment:
Could you explain why this is needed?
I'm not a fan of this pattern we've been spreading on the codebase, of passing a ton of override arguments around. It ends up generating big tangles that are hard to fix on refactors.
Can we have this use a single method, either always passing the cost as an argument or always using an object variable that stays the source of truth?

Reply:
Sure, I'll use the pack cost as argument.